server/komodo-rest/src/main/java/org/komodo/rest/KomodoRestV1Application.java

```diff
@@ -208,6 +210,11 @@
         String CONNECTIONS_SEGMENT = "connections";
 
         /**
+         * The name of the URI path segment for a setting a dataservice's service vdb
+         */
```

Comment:
This segment name is not RESTful. URIs should end in nouns not verbs while the http operation denotes the verb, eg. http://..../vdb (GET, DELETE, POST). This segment has a POST operation anyway so just drop the 'set' prefix to become "a ServiceVdbForSingleTable POST operation".